tests/components/notify/test_html5.py

```diff
@@ -1,7 +1,6 @@
 """Test HTML5 notify platform."""
 import json
-import tempfile
-from unittest.mock import patch, MagicMock
+from unittest.mock import patch, MagicMock, mock_open
```

Comment:
Yeah, it's pretty awesome, considering what a pain it can be to mock out `__builtins__`!